src/tests/integration/id/profile/id-delete-profile-date.test.ts

```diff
@@ -0,0 +1,42 @@
+import { describe, expect } from 'vitest'
+import { ID } from '../../../../id'
+import { testWithIDData } from '../test-with-id-data'
+import { ProfileDatesAttribute } from '../../../../types'
+import simpleOmneoRequest from '../../../lib/simple-omneo-request'
+import randomString from '../../../lib/string/random'
+
+const getHandle = () => { return `sdk_unit_test_dates_${randomString(5).toLowerCase()}_${Math.floor(Date.now() / 1000)}` }
+const testProfileID = process.env.OMNEO_TEST_PROFILE_ID as string
+
+describe('ID Delete Date', () => {
+  testWithIDData('ID SDK delete date', async ({ IDData }) => {
+    const { tokenData } = IDData
+    const payload: ProfileDatesAttribute = {
+      name: 'Wedding',
```

Comment:
Can you please name this "ID SDK Delete Date" 
So in case one is left behind due to some error, we know what test to fix

Reply:
changed.

Reply:
can you please change the `name` attribute on the payload. 
Apologies was not more specific